wbooks/src/app/components/TabBar/components/TabBarIcon/styles.js

```diff
@@ -0,0 +1,20 @@
+import { StyleSheet } from 'react-native';
+
+import { lightBlue, lightGray } from '../../../../../constants/colors';
+
+export default StyleSheet.create({
+    container: {
+        alignItems: 'center'
+    },
+    image: { 
+        width: 21, 
```

Comment:
extract 21 and use it here and in the next line (11)

Reply:
done!